### PR TITLE
improving class names

### PR DESCRIPTION
**Pull request by sineade**
:art:
Needed to fix some class name's that were pointed out in another PR for forms2 
https://github.com/amaabca/forms2/pull/185

### CODE REVIEW

File: app/assets/stylesheets/ama_layout/layout_components/forms.scss [hunk 1]
```diff
@@ -9,26 +9,18 @@
   float: left;
 }
 
-/* ------------- BEM Radio Buttons ------------- */
-.radio--label{
```
Comment (kaytcampbell) [inline app/assets/stylesheets/ama_layout/layout_components/forms.scss]: It looks like forms 2 is using this class, so you will have to change the class in forms 2 when you bump the gem version in there

Comment (sineade) [inline app/assets/stylesheets/ama_layout/layout_components/forms.scss]: yup, already done in my bem-forms2 branch :)

**Comment by sineade** [comment]
:wave:

**Comment by kaytcampbell** [comment]
:hankey: It looks like Matt updated ama layout, so you will have to bump the version again

**Comment by sineade** [comment]
:wave:

**Comment by kaytcampbell** [comment]
:art: :art: :art:

**Comment by sineade** [comment]
@kaytcampbell I need you to approve the changes too please

**Pull request merged**